content/03.aim1.md

```diff
@@ -56,10 +58,70 @@ The performance of VAE and t-SNE is similar and much better than PCA, but with t
 
 Figure 2: 2 - layer VAE is more resistant to outlier. 2D visualization of simulated single cell data from different outlier parameters ( 0 - 0.5).
 
-#### 3. Performance evaluation of different simulation parameters
+##### 3. Performance evaluation of different simulation parameters
+
 Clustering performance was measured by normalized mutual information (NMI).
 NMI is an normalization of the Mutual Information (MI, measures the dependence of two random variables). It is a measurement to determine the quality of clustering, which is between 0 (no mutual information) and 1 (perfect correlation).
 
 ![](images/nmi.png)
 
-Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters. 
+Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
+
+#### VAE performance on real single cell datasets
+
+##### 1. real single cell datasets:
+
+To evaluate the performance of VAE and the other dimension reduction approaches, we selected 8 real single cell datasets with true cell type labels.
+
+| dataset | ncells | ngenes | tissue | organism | cell types | Accesion number | Platform | celltype annotation |
+| ------- | ------ | ------ | ------ | -------- | --------- | --------- | ------------------ | -------------------------------------- |
+| yan | 90 | 20214 | Embryo Devel | human | 6 | GSE36552 | Illumina HiSeq 2000 | from different embryonic developmental time points |
```

Comment:
This is a new comment, but do you want to cite these papers in the table? Manubot supports references and it'd be handy to have things all in one place.

Reply:
ah, I can add citations into tables